pages/common/moreutils.md

```diff
@@ -0,0 +1,13 @@
+# moreutils
+
+> A collection of the UNIX tools that nobody thought to write long ago when UNIX was young.
```

Comment:
```suggestion
> A collection of UNIX tools.
```

Reply:
This change is necessary, otherwise the text reads like an advertisement.